otel-integration/k8s-helm/e2e-test/expected_test.go

```diff
@@ -148,10 +148,5 @@ var expectedMetrics map[string]bool = map[string]bool{
 	"otelcol_receiver_accepted_log_records":          false,
 	"otelcol_processor_batch_timeout_trigger_send":   false,
 	"otelcol_exporter_send_failed_log_records":       false,
```

Comment:
Do you think worth to mention about this metrics not being available anymore on our change log?